docs/notebooks/deep_gp_samples.py

```diff
@@ -17,23 +17,34 @@
 """
 # Deep GP samples
 
-TODO: Some explanation...
+To help develop a more intuitive understanding of deep Gaussian processes, in this notebook we show how to generate a sample from the full deep GP, by propagating a sample through the layers.
 """
 # %%
+import matplotlib.pyplot as plt
+import numpy as np
+import tensorflow as tf
+
 import gpflow
 from gpflux.helpers import construct_basic_kernel, construct_basic_inducing_variables
 from gpflux.layers import GPLayer
-import numpy as np
-import tensorflow as tf
-from gpflux.experiment_support.plotting import plot_layers
+from gpflux.experiment_support.plotting import plot_layer
 
 tf.random.set_seed(42)
 
 # %%
-Ns = 1000
+num_data = 200
 D = 1
 a, b = 0, 1
-X = np.linspace(a, b, 1000).reshape(-1, 1)
+X = np.linspace(a, b, num_data).reshape(-1, 1)
+
+# %% [markdown]
+# ## Constructing the layers
+#
+# Note that we give the `full_cov=True` argument to `GPLayer` so that we obtain correlated samples.
+# We give the last layer a `gpflow.mean_functions.Zero` mean function (the GPflux default is an Identity mean function).
```

Comment:
```suggestion
"""
## Constructing the layers

Note that we give the `full_cov=True` argument to `GPLayer` so that we obtain correlated samples.
We give the last layer a `gpflow.mean_functions.Zero` mean function (the GPflux default is an Identity mean function).
"""
```